### PR TITLE
Restore pindata header colors

### PR DESCRIPTION
**Pull request by ivov**
Restored from [tokens stylesheet](https://github.com/n8n-io/n8n/pull/3512/files#diff-9b08f3267a70b652b4317e01400d9096d3f419524638cd4327d9433129d67a9f) in pindata branch.

### CODE REVIEW

**Review by alexgrozav — APPROVED** [review]
Thanks for taking care of it!

File: packages/design-system/src/components/N8nPanelCallout/PanelCallout.stories.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ export default {
 	},
 };
 
-const template: StoryFn = (args, { argTypes }) => ({
```
Comment (alexgrozav) [inline packages/design-system/src/components/N8nPanelCallout/PanelCallout.stories.js:35]: Any reason for removing the type here?

Comment (ivov) [reply packages/design-system/src/components/N8nPanelCallout/PanelCallout.stories.js:35]: Because it's a JS file.

**Comment by janober** [comment]
Thanks a lot. Got merged.

**Comment by janober** [comment]
Got released with `n8n@0.188.0`

**Pull request merged**